src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -25,25 +25,30 @@
  */
 public class RobotPreferences {
   public static final class prefClimber {
-    public static final SN_DoublePreference climberS = new SN_DoublePreference("climberS", 0);
-    public static final SN_DoublePreference climberP = new SN_DoublePreference("climberP", 250);
-    public static final SN_DoublePreference climberI = new SN_DoublePreference("climberI", 0);
-    public static final SN_DoublePreference climberD = new SN_DoublePreference("climberD", 0);
+    public static final SN_DoublePreference climberGtele = new SN_DoublePreference("climberGtele", 0);
+    public static final SN_DoublePreference climberPtele = new SN_DoublePreference("climberPtele", 250);
+    public static final SN_DoublePreference climberItele = new SN_DoublePreference("climberItele", 0);
+    public static final SN_DoublePreference climberDtele = new SN_DoublePreference("climberDtele", 0);
+
+    public static final SN_DoublePreference climberGClimb = new SN_DoublePreference("climberGClimb", 100);
+    public static final SN_DoublePreference climberPClimb = new SN_DoublePreference("climberPClimb", 250);
+    public static final SN_DoublePreference climberIClimb = new SN_DoublePreference("climberIClimb", 0);
+    public static final SN_DoublePreference climberDClimb = new SN_DoublePreference("climberDClimb", 0);
 
     /**
      * <b>Units:</b> Degrees
      */
     public static final SN_DoublePreference climberMotorForwardLimit = new SN_DoublePreference(
-        "climberMotorForwardLimit", 85);
+        "climberMotorForwardLimit", 100);
     /**
      * <b>Units:</b> Degrees
      */
     public static final SN_DoublePreference climberMotorReverseLimit = new SN_DoublePreference(
         "climberMotorReverseLimit", 0);
     public static final SN_DoublePreference climberMotorUpSpeed = new SN_DoublePreference(
-        "climberMotorUpSpeed", 0.2);
+        "climberMotorUpSpeed", 0.6);
     public static final SN_DoublePreference climberMotorDownSpeed = new SN_DoublePreference(
-        "climberMotorDownSpeed", -0.2);
+        "climberMotorDownSpeed", -0.6);
```

Comment:
Are these used anymore now that we are using set points?

Reply:
Nope